release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -65,6 +65,16 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 * With this update, you can export your application in the ZIP file format to another project or cluster using the *Export application* option on the *+Add* page. 
 * With this update, you can create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
 
+[id="ocp-4-12-helm-page-improvements"]
+==== Helm page improvements
+
+In {product-title} 4.12, you can use the Helm page to:
+
+* Create Helm Releases and Repositories using the *Create* button.
```

Comment:
It seems like these might be lower-case terms.
```suggestion
* Create Helm releases and repositories using the *Create* button.
```

Reply:
Hi @rolfedh - These two terms are UI elements in the Helm page. Adding a PR link for reference here (https://github.com/openshift/console/pull/11844) which contains GIFs of these developed UI items. Kindly let me know if I should still keep them lower-cased.

Reply:
Thanks, Dhruv. If we're referring to a creating a release, we use lower case like this example from `modules/odc-rolling-back-helm-release.adoc`:
`If a release fails, you can rollback the Helm release to a previous version.` 
Otherwise, if we're referring to the Ui, we use bold and match the title of the page, like this example from the same file:
`. In the *Helm Releases* page, click on the chart to see the details and resources for that release.`